.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+## Status
+**READY/IN DEVELOPMENT/HOLD**
+ - [ ] Ready
+ - [ ] In development
+ - [ ] Hold
+
+## Closes issue(s)
+A list of issues that are closed by this PR
+
+## Description
+A few sentences describing the overall goals of the pull request's commits.
+
+# Changes include
+- [ ] Bugfix (non-breaking change that solves an issue)
+- [ ] New feature (non-breaking change that adds functionality)
+- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)
+- [ ] Refactor (non-breaking/breaking change that refactor the previous code base)
+
+## Impacted Areas in Application
+List general components of the application that this PR will affect:
+
+## Todos
+- [ ] Updated and Passed Tests
+- [ ] Passed Quality Checks
```

Comment:
```suggestion
- [ ] Updated Tests
```
Doesn't the CI pipeline already check the quality check and tests pass?

Reply:
The passed quality checks is basically a reminder for us to run quality checking tools locally or do some initial quality checking locally